AdminWidget.py

```diff
@@ -0,0 +1,681 @@
+                                      "_____________________________________________________________________________________________________________________________________________________________________________________________________________________________________________"))
+        self.FROMAddLabel.setPlaceholderText(_translate("ADMINPanel", "FROM"))
+        self.TOAddLabel.setPlaceholderText(_translate("ADMINPanel", "TO"))
+        self.DATEAddLabel.setPlaceholderText(_translate("ADMINPanel", "DATE"))
+        self.TIMEAddLabel.setPlaceholderText(_translate("ADMINPanel", "TIME"))
+        self.FLIGHTNUMBERAddLabel.setPlaceholderText(_translate("ADMINPanel", "FLIGHT NUMBER"))
+        self.PLANENUMBERAddFlightLabel.setPlaceholderText(_translate("ADMINPanel", "PLANE NUMBER"))
+        self.PRICEAddLabel.setPlaceholderText(_translate("ADMINPanel", "PRICE"))
+        self.ADDFLIGHTButton.setText(_translate("ADMINPanel", "ADD FLIGHT"))
+        self.FLIGHTNUMBERDeleteLabel.setPlaceholderText(_translate("ADMINPanel", "FLIGHT NUMBER"))
+        self.DELETEFLIGHTButton.setText(_translate("ADMINPanel", "DELETE FLIGHT"))
+        self.PLANE.setText(_translate("ADMINPanel", "PLANE"))
+        self.Line2.setText(_translate("ADMINPanel",
+                                      "____________________________________________________________________________________________________________________________________"))
+        self.PLANENUMBERAddPlaneLabel.setPlaceholderText(_translate("ADMINPanel", "PLANE NUMBER"))
+        self.YEAROFPRDAddLabel.setPlaceholderText(_translate("ADMINPanel", "YEAR OF PRODUCTION"))
+        self.FLIGHTSMILESAddLabel.setPlaceholderText(_translate("ADMINPanel", "FLIGHTS MILES"))
+        self.CAPACITYAddLabel.setPlaceholderText(_translate("ADMINPanel", "CAPACITY"))
+        self.ADDPLANEButton.setText(_translate("ADMINPanel", "ADD PLANE"))
+        self.PLANENUMBERDeleteLabel.setPlaceholderText(_translate("ADMINPanel", "PLANE NUMBER"))
+        self.DELETEPLANEButton.setText(_translate("ADMINPanel", "DELETE PLANE"))
+
+    def deleteFlight(self):
+        self.db = TicketAir.DataBase()
+        self.ERRORFrame.show()
+        if self.db.cancelFlight(str(self.FLIGHTNUMBERDeleteLabel.text())):
+            self.ERRORLabel.setText(f'Successfully deleted flight with number: {str(self.FLIGHTNUMBERDeleteLabel.text())}')
+            self.home.setOrigins()
+        else:
+            self.ERRORLabel.setText('Could not delete this flight.')
+
+    def deletePlane(self):
+        self.db = TicketAir.DataBase()
+        self.ERRORFrame.show()
+        if self.db.removePlane(str(self.PLANENUMBERDeleteLabel.text())):
+            self.ERRORLabel.setText(f'Successfully deleted plane with tail number: {str(self.PLANENUMBERDeleteLabel.text())}')
+            self.home.setOrigins()
+        else:
+            self.ERRORLabel.setText('There is no plane with that tail number.')
+
+    def addFlight(self):
+        try:
```

Comment:
Do you need try/except for all this code, or would try/except for only line 633/638 be sufficient?